core/src/test/java/org/apache/calcite/util/UtilTest.java

```diff
@@ -914,7 +914,7 @@ private List<Integer> makeConsList(int start, int end) {
     }
     // Example locale names in Locale.toString() javadoc.
     String[] localeNames = {
-        "en", "de_DE", "_GB", "en_US_WIN", "de__POSIX", "fr__MAC"
+        "en", "de_DE", "gb_GB", "en_US_WINDOWS", "de__POSIX", "fr__MACOS"
```

Comment:
need to extend `en_US_WIN` to `en_US_WINDOWS` since `Locale#forLanguageTag` assumes that variants contain between 4 and 8 symbols